Monika After Story/game/script-topics.rpy

```diff
@@ -1570,13 +1573,53 @@ label monika_yuri:
     m 2euc "But if you do happen to be into the yandere type..."
     show monika 5eua at t11 zorder MAS_MONIKA_Z with dissolve
     m 5eua "I can try acting a little more creepy for you. Ehehe~"
-    m "Then again..."
-    show monika 4hua at t11 zorder MAS_MONIKA_Z with dissolve
-    m 4hua "There's already nowhere else for you to go, or anyone for me to get jealous over."
-    m 2etc "Is this a yandere girl's dream?"
-    if not persistent._mas_pm_cares_about_dokis:
-        m 1eua "I'd ask Yuri if I could."
-    return
+    $ _history_list.pop()
+    menu:
+        m "I can try acting a little more creepy for you. Ehehe~"
```

Comment:
Missing `{fast}`. 

This isn't really a question, so I don't think we need a menu here.